images/awx-ee/bindep.txt

```diff
@@ -8,6 +8,7 @@ php-zip
 php-mbstring
 php-phar
 php-soap
+php-sodium
```

Comment:
https://archlinux.org/packages/extra/x86_64/php-sodium/

This looks correct to me, unless it's using some kind of weird packaging system on the underlying awx platform.